src/inference/tests/unit/ie_extension_test.cpp

```diff
@@ -17,6 +17,8 @@ using namespace InferenceEngine;
 
 using ExtensionTests = ::testing::Test;
 
+std::string getExtensionPath();
```

Comment:
Why do you need this forward declaration?

Reply:
The same comment as before....
I don't believe that we need declaration before definition